### DISCUSSION TITLE
Kdenlive

### DISCUSSION THREAD

**Discussion by Merlimau** [mayrinck/FOSSonMicrosoftStore#23] [General]
Hi Mayrink

Thank you for bringing Kdenlive to the Microsoft store. I'm from Kdenlive core team. I see some differences in download size, number of files and folders when Kdenlive 21.04.2 is installed. Which source do you have taken to build Kdenlive?

We have been planning to put Kdenlive in the Microsoft store for a while but we feel that we are not ready yet to publish it as there are some tasks we need to resolve prior to that, some are technical and others administrative. We kindly ask you to remove Kdenlive from the Microsoft store but would like to invite you to join the effort and help us release it officially when we feel it is ready for prime time.

Eugen Mohr AKA Merlimau

**Comment by JuliaAgostino** [comment]
I assume the download size and numbers of files and folders are different because the app has been packaged, but I could be wrong.

I'm glad to hear that Kdenlive will eventually come to the Microsoft Store from an official source, but it's sad to see it go for now. I'm glad this repository is at least getting some of the recognition it deserves.

**Comment by mayrinck** [comment]
Hello @Merlimau, I downloaded the installer from the website and used MSIX package tool to convert it to the MS Store... As part of our project, we never mess with the code of any app. We will put down the app from the store as soon as you prove your identity and/or upload yours there... See ya!

**Comment by Merlimau** [comment]
Hello @mayrinck. Back from vacation. Thank you for the feedback. How can I prove my identity? You see my name in Kdenlive/about -> tab "Thanks to" (Eugen Mohr). Check on www.Kdenlive.org -> Forum (Merlimau: https://forum.kde.org/viewtopic.php?f=265&t=171856). 

We from Kdenlive team would appreciate if you could upload Kdenlive in the Microsoft store in the future.

EDIT: I created an issue on GitLab (home of Kdenlive's code base) for proving my identity: https://invent.kde.org/multimedia/kdenlive/-/issues/1133

**Reply by mayrinck** [reply]
Okay then, I will remove kdenlive from the store, the graphic materials and other stuff we used to publish there will be still accessible on this repo okay? If you guys ever need it, you can pick them here os ask me them here on Github :)
